content/en/docs/setup/install/multicluster/shared-gateways/index.md

```diff
@@ -177,6 +177,7 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     {{< text bash >}}
     $ kubectl create --context=$CTX_CLUSTER2 ns istio-system
     $ kubectl create --context=$CTX_CLUSTER2 secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
+    $ CLUSTER_NAME=$(kubectl --context=$CTX_CLUSTER2 config view --minify=true -o jsonpath='{.clusters[].name}')
```

Comment:
CTX_CLUSTER2 is the cluster name already in https://preliminary.istio.io/docs/setup/install/multicluster/shared-gateways/#prerequisites:
```
export CTX_CLUSTER2=$(kubectl config view -o jsonpath='{.contexts[1].name}')
```

Reply:
@frankbu I'm not sure that's always true anymore. I did a quick check on IKS where I now rename the context because the context name is now longer.  Thus my context name might be istio-1 but the cluster name is istio-1/abcd...

I know I have run through this scenario many times on IKS (but not sure how many times after IKS made the context name change) so not sure of the actual effect on this.

Reply:
Ah, I see, my mistake ... "cluster" name vs "context" name